src/main/resources/stairway/db/changesets/20210416_flightworking.yaml

```diff
@@ -0,0 +1,49 @@
+databaseChangeLog:
+  - changeSet:
+      id: flightworking
+      author: JCz
+      changes:
+        - dropNotNullConstraint:
+            tableName: flightlog
+            columnName: working_parameters
+            remarks: deprecating for flightworking table
+        - addColumn:
+            tableName: flightlog
+            columns:
+              - column:
+                  name: id
+                  type: uuid
+                  constraints:
```

Comment:
Postrgres cannot generate default UUID's without a plugin, so we cannot upgrade with a non-nullable constraint.  I did run all tests with a non-null constraint in place to try to catch non-null inserts in code.